src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx

```diff
@@ -3,7 +3,7 @@ path: '/build-apps/publish-deploy/catalog'
 title: "Update your Nerdpack's catalog information"
```

Comment:
Anyone know why this URL has a 404? 

https://developerwebsitedevelop-mlosieriopackdetailsrefresh.gtsb.io/build-apps/publish-deploy/catalog

Live site seems fine: https://developer.newrelic.com/build-apps/publish-deploy/catalog/